packages/angular/cli/lib/config/schema.json

```diff
@@ -835,6 +835,11 @@
             "ngswConfigPath": {
               "type": "string",
               "description": "Path to ngsw-config.json."
+            },  
+            "autoBundleWorkerModules": {
```

Comment:
Maybe we can change the name? Not very keen on this one.

Reply:
Igor also wasn't keen on the name, so I image we will change it.

We also need an option for the worker tsconfig.

Reply:
Suggestion: `bundleModuleWorkers`